vm/src/stdlib/os.rs

```diff
@@ -2371,6 +2372,10 @@ mod posix {
     #[pyfunction]
     fn openpty(vm: &VirtualMachine) -> PyResult {
         let r = nix::pty::openpty(None, None).map_err(|err| err.into_pyexception(vm))?;
+        for fd in &[r.master, r.slave] {
+            fcntl::fcntl(*fd, fcntl::F_SETFD(fcntl::FdFlag::FD_CLOEXEC))
```

Comment:
Not really versed in `os` stuff but, I see there's `raw_set_inheritable` over on line 2071. Wouldn't that do the trick here?

Reply:
Yep, that'd be good to use for this @deantv

Reply:
Yes, I should use this. Fixed in new commit.